### PR TITLE
Buffs the particle decelerator

### PR DESCRIPTION
**Pull request by Flareguy**
## About the PR
Buffs the particle decelerator to be 6.5x stronger (-10 energy to -65 energy.)
In theory, this is still not good enough to do much with 1 player firing, but makes teams of players actually useful.

Also adds debug prototypes for buffed particle decelerators, to simulate multiple players.

## Why / Balance
-10 singularity energy is ridiculously terrible - you'd need a team of 10+ people to even put a dent in a low level singularity. This should make particle decelerators actually usable for situations outside of admin-spawned ERTs.

## Media
- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**
- tweak: Particle decelerators are now significantly stronger.

### CODE REVIEW

**Comment by Brandon-Huu** [comment]
>  A single particle decelerator can now roughly shrink the singularity by 1 stage, or almost 1 stage.
That's awful.

one person shouldn't have that much power

**Comment by EmoGarbage404** [comment]
I know it shouldnt be impossible, but decels are 3 a piece in cargo and have 20 shots.
This current iteration feels extremely strong, especially for early stage singulos, where a single person with a decel can disable a singulo from outside of engineering in just a few seconds

**Comment by Flareguy** [comment]
Nerfed it down to -65 SinguloFood. This is probably a bit of a lowball and, from what I saw, was able to get a level 6 singulo down to either level 3 or level 4 with 3 people's worth of firerate & ammo simulated.

I don't remember how it fared with 1 player, but it was not very well.

**Review by ShadowCommander — APPROVED** [review]

**Comment by osjarw** [comment]
How does this work with tesla? 
IIRC, tesla handles singulofood differently, that is, it varies between -100 and 100, hitting 100 sets it back to 0 while spawning a small energy ball, and if decelerators get it to -100, the tesla dies.
So with singulofood -65, you can kill the tesla in anywhere between 2-4 shots if the tesla isn't actively consuming stuff.

**Comment by ShadowCommander** [comment]
One suggestion that was brought up is to make the particle decelerator percentage based so that it can still kill level 5 and level 6 singularities without seemingly instakilling level 3 and below singularities.

**Comment by Ilya246** [comment]
why not just have it deal flat *and* percentage damage together so it works well against high-health singularities but can still kill low-health ones fast

**Comment by wafehling** [comment]
We need this change so badly. Half, possibly more of rounds (In the last couple weeks, in my personal accounting) now end from singuloose/tesloose, and it's rare that anyone tries to counter-play them. I've seen people try several times, and I've never seen the particle decels succeed once against them, even with several dedicated users trying.

File: Resources/Prototypes/Entities/Objects/Weapons/Guns/Battery/battery_guns.yml [hunk 1]
```diff
@@ -378,6 +378,34 @@
...
+- type: entity
+  name: portable particle decelerator x2
+  suffix: DEBUG
+  parent: WeaponParticleDecelerator
+  id: WeaponParticleDeceleratorX2
+  description: A particle decelerator that simulates 2 people firing them at once.
+  components:
+    - type: Gun
+      fireRate: 1
+      fireCost: 500
```
Comment (slarticodefast) [inline Resources/Prototypes/Entities/Objects/Weapons/Guns/Battery/battery_guns.yml:390]: remove fireCost to stop the yaml linter complaining. This is a part of ProjectileBatteryAmmoProvider and not changed anyways.
```suggestion
```

File: Resources/Prototypes/Entities/Objects/Weapons/Guns/Battery/battery_guns.yml [hunk 2]
```diff
@@ -378,6 +378,34 @@
...
+- type: entity
+  name: portable particle decelerator x3
+  suffix: DEBUG
+  parent: BaseWeaponBattery
+  id: WeaponParticleDeceleratorX3
+  description: A particle decelerator that simulates 3 people firing them at once.
+  components:
+    - type: Gun
+      fireRate: 1.5
+      fireCost: 500
```
Comment (slarticodefast) [inline Resources/Prototypes/Entities/Objects/Weapons/Guns/Battery/battery_guns.yml:404]:
```suggestion
```

File: Resources/Prototypes/Entities/Objects/Weapons/Guns/Battery/battery_guns.yml [hunk 3]
```diff
@@ -378,6 +378,34 @@
...
       startingCharge: 10000
 
+- type: entity
+  name: portable particle decelerator x2
+  suffix: DEBUG
+  parent: WeaponParticleDecelerator
+  id: WeaponParticleDeceleratorX2
+  description: A particle decelerator that simulates 2 people firing them at once.
+  components:
+    - type: Gun
```
Comment (slarticodefast) [inline Resources/Prototypes/Entities/Objects/Weapons/Guns/Battery/battery_guns.yml:388]: Please fix the indentation for the components. Do the same with the WeaponParticleDecelerator, it is wrong there, too.

**Review by slarticodefast — CHANGES_REQUESTED** [review]

**Comment by TheShuEd** [comment]
do you plan to continue your work?

**Comment by TheShuEd** [comment]
summarizing internal discussions: damage should be a percentage, not flat. A different implementation is required. Given that the PR is outdated, reviews are not fixed, and checks are not passed, we are closing the PR

**Pull request closed without merging**